query-engine/query-engine-node-api/src/engine.rs

```diff
@@ -201,7 +201,9 @@ impl QueryEngine {
 
     /// Connect to the database, allow queries to be run.
     #[napi]
-    pub async fn connect(&self) -> napi::Result<()> {
+    pub async fn connect(&self, trace: String) -> napi::Result<()> {
```

Comment:
```suggestion
    pub async fn connect(&self, trace?: String) -> napi::Result<()> {
```

I know using `?` is not the correct way, however, do we need to make this arg and the disconnect arg optional @garrensmith. The arg will only be sent if tracing is enabled.